VMBackup/main/handle.py

```diff
@@ -165,33 +164,41 @@ def snapshot():
         errMsg = 'Failed to do the snapshot with error: %s, stack trace: %s' % (str(e), traceback.format_exc())
         backup_logger.log(errMsg, False, 'Error')
     snapshot_done = True
-
-
-def freeze_watcher():
-    global backup_logger,run_status,error_msg,snapshot_done
-    #wait 5 minutes before the snapshoting.
-    try:
-        for i in range(0, 50):
-            if(snapshot_done):
-                backup_logger.log('T:W snapshot is done', False)
-                break;
-            sleep(6)
-        if not snapshot_done:
-            run_result = CommonVariables.error
-            run_status = 'error'
-            error_msg = 'T:W Snapshot timeout'
-            backup_logger.log(error_msg, False, 'Warning')
-    except Exception as e:
-        errMsg = 'Failed to do  the snapshot because of exception in freeze watcher'
-        backup_logger.log(errMsg, False, 'Error')
 def daemon():
-    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser
+    global MyPatching,backup_logger,hutil,run_result,run_status,error_msg,freezer,para_parser,snapshot_done
     #this is using the most recent file timestamp.
     hutil.do_parse_context('Executing')
     freezer = FsFreezer(patching= MyPatching, logger = backup_logger)
     global_error_result = None
     # precheck
     freeze_called = False
+
+    configfile='/var/lib/waagent/extension_config/config.ini'
```

Comment:
@soumyanishan I don't think `extension_config` usually won't mean anything to anyone, and you're already in /etc, meaning it's already a config. How about: `/etc/azure/vmbackup.conf` (or `.cfg`).

`.ini` is a Windows convention, let's not bring it to Linux. 😉